widgetastic_manageiq/__init__.py

```diff
@@ -5141,3 +5141,63 @@ def click_body_action(self):
 
     def click(self):
         return self.click_title()
+
+
+class MultiBoxOrderedSelect(MultiBoxSelect):
```

Comment:
Question : There is already a widget called MultiBoxSelect . 
Can you modify that  to your requirements ?

Reply:
Yes, I am modifying `MultiBoxSelect`... Its not fallowing `sequence` of values... So its different than that widget. It will take care of `Order` of values... pointing index of values.

Reply:
:+1: